frontend/src/v5/ui/components/viewer/cards/card.styles.ts

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
I am not sure about this, but this file is the `.styles.ts` file for `tickets.tsx`, but it is named differently and located in a different directory. Is it on purpose?

Reply:
This is for cards in the viewer for v5. Is not tickets specific.